doc/sphinx-guides/source/developers/sql-upgrade-scripts.rst

```diff
@@ -2,40 +2,38 @@
 SQL Upgrade Scripts
 ===================
 
-The database schema for Dataverse is constantly evolving. As other developers make changes to the database schema you will need to keep up with these changes to have your development environment in working order. Additionally, as you make changes to the database schema, you must write SQL upgrade scripts when needed and communicate with your fellow developers about applying those scripts.
+The database schema for Dataverse is constantly evolving. As other developers make changes to the database schema you will need to keep up with these changes to have your development environment in working order. Additionally, as you make changes to the database schema, you must write SQL when needed and communicate with your fellow developers about making those SQLs.
 
 .. contents:: |toctitle|
 	:local:
 
-Location of SQL Upgrade Scripts
--------------------------------
+Location of SQL Upgrade Scripts and Flyway SQLs
+-----------------------------------------------
 
-``scripts/database/upgrades`` is the directory where we keep or SQL upgrade scripts.
+``scripts/database/upgrades`` is the directory where we keep SQL upgrade scripts - legacy way of upgrading schema.
+``resources/db/migration`` is the directory where we keep migration SQLs.
 
-How to Determine if You Need to Create or Update a SQL Upgrade Script
----------------------------------------------------------------------
+SQL naming conventions
+----------------------
+In order to make the migrations work you must follow the naming conventions of the flyway - https://flywaydb.org/documentation/migrations#sql-based-migrations
 
-If you are creating a new database table (which maps to an ``@Entity`` in JPA), you do not need to create or update a SQL upgrade script. The reason for this is that we use ``create-tables`` in ``src/main/resources/META-INF/persistence.xml`` so that new tables are automatically created by Glassfish when you deploy your war file.
+How to Determine if You Need to Create SQL
+------------------------------------------
 
-If you are doing anything other than creating a new database table such as adding a column to an existing table, you must create or update a SQL upgrade script.
+If you are creating a new database table (which maps to an ``@Entity`` in JPA), you do not need to create. The reason for this is that we use ``create-tables`` in ``src/main/resources/META-INF/persistence.xml`` so that new tables are automatically created by Glassfish when you deploy your war file.
 
-How to Create or Update a SQL Upgrade Script
---------------------------------------------
+If you are doing anything other than creating a new database table such as adding a column to an existing table, you must create SQL.
```

Comment:
Maybe we should place a hint here that adding a value to an `@Entity` is just the same?
Maybe a practical example is usefull here, too.

Reply:
I'm not exactly sure what do you mean by the same, adding a field in `@Entity` annotated class without any SQL script wouldn't work since our DDL Generetion is set to `create-tables` only.